vaadin-tabs-flow-parent/vaadin-tabs-flow/src/test/java/com/vaadin/flow/component/tabs/tests/TabsTest.java

```diff
@@ -205,4 +206,13 @@ public void removeTabInTabsWithoutSelection() {
         Assert.assertNull("should not change selected tab",
                 tabs.getSelectedTab());
     }
+
+    @Test
+    public void addTabsAsArrayDoesNotCauseClassCastException() {
```

Comment:
Maybe the naming can be more specific like `addTabsAsComponentArray_noClassCastExceptionIsThrown `

Reply:
Will do, thanks